RELEASING.md

```diff
@@ -14,14 +14,43 @@ The following are step-by-step instructions for releasing the Promscale extensio
 - [ ] Tag and push merged release commit.
   - CI will trigger and create a draft release with assets attached.
 - [ ] Prepare extension release notes, share with team asking for feedback.
-- [ ] Manually tag and push `timescaledev/promscale-extension` docker image (see #354)
 - [ ] Attach release notes to draft release created above.
-- [ ] Publish the GitHub release
+- [ ] Publish the GitHub release.
 
 ## Post-Release
 - [ ] Prepare for the next development cycle:
   - Generate SQL schema and commit it (and upgrade symlinks to it) to the `sql` directory.
+<details>
```

Comment:
I would prefer it if we would keep the original formatting of this file as a bullet point checklist. If something can't be done as a bullet point item, then we should add some tooling to allow this to be the case.